internal/rbd/controllerserver.go

```diff
@@ -1050,6 +1050,13 @@ func (cs *ControllerServer) ControllerExpandVolume(ctx context.Context, req *csi
 		return nil, status.Error(codes.InvalidArgument, "capacityRange cannot be empty")
 	}
 
+	// Get the nodeexpansion flag set based on the volume mode
```

Comment:
Is there really no need to do expansion on the node when using a block PV? At the very least, the block-device needs re-scanning so that the new size is known on the node. I wonder if that needs `nodeExpansion` or some flag like that.

Reply:
True.. it need confirmation. That said, the mapped device should reflect, considering we declare ONLINE, I believe this will go through.. but good to validate it too.

Reply:
@nixpanic all the tests pass, so we are safe. :+1: